android-todo-list/architecture-samples-main/app/src/main/java/com/statsig/todoapp/tasks/TasksScreen.kt

```diff
@@ -97,6 +103,31 @@ fun TasksScreen(
     ) { paddingValues ->
         val uiState by viewModel.uiState.collectAsStateWithLifecycle()
 
+        val bannerDynConf = Statsig.getConfig("warning_banner")
+        val bannerWarningMsg =
+            bannerDynConf.getString("message", "NA") ?: stringResource(R.string.not_applicable)
```

Comment:
make this a nullable string, and then just check for null

Reply:
Okay, I will change it to this. And in case of null values do I need to show some default banner or nothing at all is fine?